src/Hamiltonians/ExtendedHubbardReal1D.jl

```diff
@@ -13,20 +13,45 @@ Implements the extended Hubbard model on a one-dimensional chain in real space.
 * `u`: on-site interaction parameter
 * `v`: the next-neighbor interaction
 * `t`: the hopping strength
+* `boundary_condition` : applied boundray condition 
```

Comment:
```suggestion
* `boundary_condition` : applied boundary condition 
```
In addition to fixing this typo, can you please write the documentation for the possible boundary conditions as you did for the `hopnextneighbour` function?

Reply:
I have mentioned all the possible boundary conditions below that line. I guess I did not make it clear before, but now I have made the essential changes; please look at it once.

Reply:
Also, I mentioned in the Zoom meeting that I am working on another PR and still need time to post.

Reply:
OK I'll wait for the other PR.

With the comment about the docstring of `ExtendedHubbardReal1D` I meant to suggest to rephrase it consistent with the suggestion I had made previously for `hopnextneighbour`. In particular I meant to rephrase the description of passing a number for the keyword argument as well as the way the exponential function is typeset. Apologies for not expressing myself clearly! I'll post a concrete suggestion for rewording the docstring below.